src/overlays/actors/ovl_En_Box/z_en_box.c

```diff
@@ -4,14 +4,64 @@
 
 #define THIS ((EnBox*)thisx)
 
+typedef enum {
+    /*
+     set on init unless treasure flag is set
+     if clear, chest moves (Actor_MoveForward) (falls, likely)
+     ends up cleared from SWITCH_FLAG_FALL types when switch flag is set
+    */
+    ENBOX_MOVE_IMMOBILE = 1 << 0,
+    /*
+     set in the logic for SWITCH_FLAG_FALL types
+     otherwise unused
+    */
+    ENBOX_MOVE_UNUSED = 1 << 1,
+    /*
+     set with 50% chance on init for SWITCH_FLAG_FALL types
+     only used for SWITCH_FLAG_FALL types
+     ends up "blinking" (set/clear every frame) once switch flag is set,
+        if some collision-related condition (?) is met
+     only used for signum of z rotation
+    */
+    ENBOX_MOVE_FALL_ANGLE_SIDE = 1 << 2,
+    /*
+     when set, gets cleared next EnBox_Update call and clip to the floor
+    */
+    ENBOX_MOVE_STICK_TO_GROUND = 1 << 4
+} EnBox_MovementFlags;
```

Comment:
We don't really have any named flags right now so there is no standard, but imho flags should be defines, not enums with hardcoded values. It's also how decomp projects normally do it, or even just C codebases in general afaik.